newsfragments/5693.bugfix.rst

```diff
@@ -0,0 +1 @@
+Fix file creation/modification times as shown in the file explorer on Windows
```

Comment:
Maybe explain a little bit the bug? Even if there is a link to the issue, it it is always nice to read a changelog that briefly explain what has been fixed :)

  ```suggestion
Fix file creation/modification time displayed in local time instead of UTC in the Windows File Explorer
```

Reply:
I understand. Maybe something like this to avoid going into too much details:
```suggestion
Fix file creation/modification times wrongly shifted as shown in the file explorer on Windows
```
```suggestion
Fix wrong offset in file creation/modification times as shown in the file explorer on Windows
```
Anyway, it's just a suggestion, you're free to take it or not ;)